sympy/functions/elementary/exponential.py

```diff
@@ -1010,20 +1010,20 @@ def coeff_exp(term, x):
 
         # TODO new and probably slow
         try:
-            a, b = z.leadterm(t)
-            s = z.nseries(t, n=n+b, logx=logx)
+            a, b = z.leadterm(t, cdir=1)
+            s = z.nseries(t, n=n+b, logx=logx, cdir=1)
```

Comment:
We could probably use the private `_eval_nseries` as the arguments have been sympified.

Reply:
Sure , will try it out !